mmv1/products/compute/Subnetwork.yaml

```diff
@@ -146,6 +146,15 @@ examples:
     vars:
       subnetwork_name: 'subnet-ipv6-only'
       network_name: 'network-ipv6-only'
+  - name: 'subnetwork_ipcollection'
+    primary_resource_id: 'subnetwork-ip-collection'
+    min_version: 'beta'
+    vars:
+      subnetwork_name: 'subnet-ip-collection'
+      network_name: 'website-net'
+      ip_collection_url: '"projects/tf-static-byoip/regions/us-central1/publicDelegatedPrefixes/tf-test-forwarding-rule-mode-pdp"'
+    test_vars_overrides:
+      ip_collection_url: '"projects/tf-static-byoip/regions/us-central1/publicDelegatedPrefixes/tf-test-forwarding-rule-mode-pdp"'
```

Comment:
Instead of using the var `ip_collection_url`, you can put the value into the example file.

Reply:
Changes applied